generators/client/templates/angular/src/main/webapp/app/admin/user-management/_user-management.component.html

```diff
@@ -78,9 +78,9 @@ <h2 jhi-translate="userManagement.home.title">Users</h2>
         </table>
     </div>
     <%_ if (databaseType !== 'cassandra') { _%>
-    <div class="text-xs-center">
+    <div class="text-xs-center" *ngIf="users">
         <jhi-item-count [page]="page" [total]="queryCount" [items-per-page]="itemsPerPage"></jhi-item-count>
-        <ngb-pagination size="sm" [collectionSize]="totalItems" [(page)]="page" [pageSize]="itemsPerPage" (click)="loadPage(page)"></ngb-pagination>
```

Comment:
`size="sm"` is removed here, which is a change to the UI. Unless this is an obsolete/unused attribute I would leave it in.